src/app/pages/adoption/adoption.html

```diff
@@ -47,7 +47,7 @@
                     </label>
                     <label>
                         <span>Email address</span>
-                        <input name="email" pattern="[^@]+@([^@\.]+\.)+[^@\.]+" type="text" required>
+                        <input name="email" pattern="[^@]+@([^@\.]+\.)+[^@\.]+" type="email" required>
```

Comment:
You actually don't need the pattern anymore. It's not supported with type `email` and no browser we support will be falling back to treating it as type `text`.